<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>qianchengdai_test_report</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="http://libs.baidu.com/bootstrap/3.0.3/css/bootstrap.min.css" rel="stylesheet">
    <script src="http://libs.baidu.com/jquery/2.0.0/jquery.min.js"></script>
    <script src="http://libs.baidu.com/bootstrap/3.0.3/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 120%; }
table       { font-size: 100%; }
/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}
.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}
/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();
/*level 调整增加只显示通过用例的分类 --Findyou
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }
    //加入【详细】切换文字变化 --Findyou
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}
function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线 -Findyou
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 --Findyou
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}
function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">qianchengdai_test_report</h1>
<p class='attribute'><strong>测试人员 : </strong> YOYO</p>
<p class='attribute'><strong>开始时间 : </strong> 2020-03-01 17:59:51</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:00.085774</p>
<p class='attribute'><strong>测试结果 : </strong> 共 30，错误 30，通过率= 0.00%</p>

<p class='description'></p>
</div>


<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>概要{ 0.00% }</a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败{ 0 }</a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过{ 0 }</a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有{ 30 }</a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 16px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_qianchengdai.test_case.test_case.TestLogin</td>
    <td class="text-center">30</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">30</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',30)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='ft1_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse in" align="left">
    <pre>
    
ft1_1: 开始执行第1条用例：正常登陆功能测试!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_2'>错误</button>
    <div id='div_ft1_2' class="collapse in" align="left">
    <pre>
    
ft1_2: 开始执行第2条用例：不输入手机号!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_3'>错误</button>
    <div id='div_ft1_3' class="collapse in" align="left">
    <pre>
    
ft1_3: 开始执行第3条用例：不输入密码!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_04</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_4'>错误</button>
    <div id='div_ft1_4' class="collapse in" align="left">
    <pre>
    
ft1_4: 开始执行第4条用例：密码输入错误!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_5' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_05</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_5' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_5'>错误</button>
    <div id='div_ft1_5' class="collapse in" align="left">
    <pre>
    
ft1_5: 开始执行第5条用例：手机号输入错误!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_6' class='none'>
    <td class='errorCase'><div class='testcase'>test_login_06</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_6' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_6'>错误</button>
    <div id='div_ft1_6' class="collapse in" align="left">
    <pre>
    
ft1_6: 开始执行第6条用例：手机号密码都为空!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 50, in test_login
    r = requests.post(url=login_url, headers=headers, data=login_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_7' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_7' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_7'>错误</button>
    <div id='div_ft1_7' class="collapse in" align="left">
    <pre>
    
ft1_7: 开始执行第1条用例：正常充值测试!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_8' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_8' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_8'>错误</button>
    <div id='div_ft1_8' class="collapse in" align="left">
    <pre>
    
ft1_8: 开始执行第2条用例：充值0元!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_9' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_9' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_9'>错误</button>
    <div id='div_ft1_9' class="collapse in" align="left">
    <pre>
    
ft1_9: 开始执行第3条用例：充值50万!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_10' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_04</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_10' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_10'>错误</button>
    <div id='div_ft1_10' class="collapse in" align="left">
    <pre>
    
ft1_10: 开始执行第4条用例：充值0.01元!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_11' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_05</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_11' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_11'>错误</button>
    <div id='div_ft1_11' class="collapse in" align="left">
    <pre>
    
ft1_11: 开始执行第5条用例：充值490000.99元!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_12' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_06</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_12' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_12'>错误</button>
    <div id='div_ft1_12' class="collapse in" align="left">
    <pre>
    
ft1_12: 开始执行第6条用例：充值-0.01元!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_13' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_07</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_13' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_13'>错误</button>
    <div id='div_ft1_13' class="collapse in" align="left">
    <pre>
    
ft1_13: 开始执行第7条用例：充值500000.01元!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_14' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_08</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_14' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_14'>错误</button>
    <div id='div_ft1_14' class="collapse in" align="left">
    <pre>
    
ft1_14: 开始执行第8条用例：不输入手机号!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_15' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_09</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_15' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_15'>错误</button>
    <div id='div_ft1_15' class="collapse in" align="left">
    <pre>
    
ft1_15: 开始执行第9条用例：手机号输入错误!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_16' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_16' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_16'>错误</button>
    <div id='div_ft1_16' class="collapse in" align="left">
    <pre>
    
ft1_16: 开始执行第10条用例：不输入金额!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_17' class='none'>
    <td class='errorCase'><div class='testcase'>test_recharge_11</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_17' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_17'>错误</button>
    <div id='div_ft1_17' class="collapse in" align="left">
    <pre>
    
ft1_17: 开始执行第11条用例：金额输入包含符号!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 67, in test_recharge
    r = requests.post(url=recharge_url, headers=headers, data=recharge_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_18' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_01</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_18' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_18'>错误</button>
    <div id='div_ft1_18' class="collapse in" align="left">
    <pre>
    
ft1_18: 开始执行第1条用例：正常注册（密码输入6位）!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_19' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_02</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_19' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_19'>错误</button>
    <div id='div_ft1_19' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_19' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_19'>错误</button>
    <div id='div_ft1_19' class="collapse in" align="left">
    <pre>
    
ft1_19: 开始执行第2条用例：不输入注册名（该项可选）!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_20' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_03</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_20' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_20'>错误</button>
    <div id='div_ft1_20' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_20' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_20'>错误</button>
    <div id='div_ft1_20' class="collapse in" align="left">
    <pre>
    
ft1_20: 开始执行第3条用例：密码输入18位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_21' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_04</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_21' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_21'>错误</button>
    <div id='div_ft1_21' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_21' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_21'>错误</button>
    <div id='div_ft1_21' class="collapse in" align="left">
    <pre>
    
ft1_21: 开始执行第4条用例：密码输入17位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_22' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_05</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_22' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_22'>错误</button>
    <div id='div_ft1_22' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_22' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_22'>错误</button>
    <div id='div_ft1_22' class="collapse in" align="left">
    <pre>
    
ft1_22: 开始执行第5条用例：密码输入7位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_23' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_06</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_23' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_23'>错误</button>
    <div id='div_ft1_23' class="collapse in" align="left">
    <pre>
    
ft1_23: 开始执行第6条用例：不输入密码!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_24' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_07</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_24' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_24'>错误</button>
    <div id='div_ft1_24' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_24' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_24'>错误</button>
    <div id='div_ft1_24' class="collapse in" align="left">
    <pre>
    
ft1_24: 开始执行第7条用例：不输入手机号!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_25' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_08</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_25' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_25'>错误</button>
    <div id='div_ft1_25' class="collapse in" align="left">
    <pre>
    
ft1_25: 开始执行第8条用例：手机号位数只有10位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_26' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_09</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_26' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_26'>错误</button>
    <div id='div_ft1_26' class="collapse in" align="left">
    <pre>
    
ft1_26: 开始执行第9条用例：手机号位数有12位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_27' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_10</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_27' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_27'>错误</button>
    <div id='div_ft1_27' class="collapse in" align="left">
    <pre>
    
ft1_27: 开始执行第10条用例：手机号格式错误!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_28' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_11</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_28' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_28'>错误</button>
    <div id='div_ft1_28' class="collapse in" align="left">
    <pre>
    
ft1_28: 开始执行第11条用例：密码只输入5位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_29' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_12</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_29' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_29'>错误</button>
    <div id='div_ft1_29' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_29' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_29'>错误</button>
    <div id='div_ft1_29' class="collapse in" align="left">
    <pre>
    
ft1_29: 开始执行第12条用例：密码输入19位!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='ft1_30' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_13</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 -Findyou
    <button id='btn_ft1_30' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_30'>错误</button>
    <div id='div_ft1_30' class="collapse">  -->
    <!-- 默认展开错误信息 -Findyou -->
    <button id='btn_ft1_30' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_30'>错误</button>
    <div id='div_ft1_30' class="collapse in" align="left">
    <pre>
    
ft1_30: 开始执行第13条用例：输入已经注册过的号码!!!
Traceback (most recent call last):
  File "D:\soft\Python\Python38\lib\site-packages\ddt.py", line 145, in wrapper
    return func(self, *args, **kwargs)
  File "F:\桌面\PycharmProjects\ningmengban_API_test.git\test_qianchengdai\test_case\test_case.py", line 33, in test_register
    r = requests.post(url=register_url, headers=headers, data=register_case.data)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 112, in post
    return request('post', url, data=data, json=json, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 498, in request
    prep = self.prepare_request(req)
  File "D:\soft\Python\Python38\lib\site-packages\requests\sessions.py", line 431, in prepare_request
    p.prepare(
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 310, in prepare
    self.prepare_headers(headers)
  File "D:\soft\Python\Python38\lib\site-packages\requests\models.py", line 442, in prepare_headers
    for header in headers.items():
AttributeError: 'str' object has no attribute 'items'


    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>30</td>
    <td>0</td>
    <td>0</td>
    <td>30</td>
    <td>通过率：0.00%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    
</body>
</html>
